src/components/bankaccounts/BankAccountsEditForm.tsx

```diff
@@ -73,34 +75,50 @@ export default function BankAccountsEditForm({ id }: Props) {
         validationSchema={validationSchemaBankAccForm}>
         <Grid container spacing={3}>
           <Grid item xs={12}>
-            <FormTextField type="text" label="bankaccounts:status" name="status" />
+            <FormSelectField
+              label="bankaccounts:fields.status"
+              name="status"
+              options={Object.keys(BankAccountStatus).map((key: string) => {
+                return {
+                  key,
+                  value: BankAccountStatus[key as BankAccountStatus],
+                  name: t(`bankaccounts:status.${key}`),
+                }
+              })}
+            />
```

Comment:
Let's extract this field as a separate reusable `BankAccountStatusSelect` component.

If we extract it will be quite simple to reuse it in both `BankAccountsEditFrom` and `BankAccountsForm` forms


```tsx
type Props = {
  name?: string
}
export default function BankAccountStatusSelect({ name = 'status' }: Props) {
  const { t } = useTranslation()
  return (
    <FormSelectField
      name={name}
      label="bankaccounts:fields.status"
      options={Object.keys(BankAccountStatus).map((key: string) => {
        return {
          key,
          value: BankAccountStatus[key as BankAccountStatus],
          name: t(`bankaccounts:status.${key}`),
        }
      })}
    />
  )
}
```

I suggest that we add the `name` prop explicitly so it's more reusable in different forms. Sometimes we'll need it as `status` but other times we might need the field name to be `bank-account-status` for example.